pom.xml

```diff
@@ -44,18 +44,13 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter</artifactId>
-      <version>3.0.3</version>
+      <version>3.0.2</version>
```

Comment:
this one as well

Reply:
@adongare This was revert back to its previous version before Ethan's change.